test/ssh.js

```diff
@@ -2,11 +2,16 @@
 
 var SSH = require('../lib/ssh.js'),
     expect = require('expect.js'),
+    fs = require('fs'),
 
     config = require('../config/ssh.json'),
     ssh;
 
 describe('SSH', function() {
+    if (config.key) {
+        config.key = fs.readFileSync(config.key);
```

Comment:
I get that but before the idea was that the key was saved directly in the config file. I suppose it makes more sense to point it to a key file though.